.teamcity/_self/builds/WPComPlugins_EditorToolkit.kt

```diff
@@ -114,27 +102,59 @@ object WPComPlugins_EditorToolKit : BuildType({
 				yarn lint:php
 			"""
 		}
+		/**
+		 * We download the archive directly in this step rather than relying on
+		 * an artifact dependency. We do this because if two commits on trunk
```

Comment:
The comment provides a super clear explanation, thanks for keeping inline documentation in mind, I appreciate that.